install_ffmpeg.sh

```diff
@@ -172,9 +168,16 @@ PATH="$HOME/bin:$PATH" PKG_CONFIG_PATH="$HOME/ffmpeg_build/lib/pkgconfig" ./conf
   --enable-nonfree \
   --enable-version3 \
   --enable-libvmaf
-PATH="$HOME/bin:$PATH" make
+PATH="$HOME/bin:$PATH" make -j 4
```

Comment:
Can u assume that you have always 4 cores?

Maybe using 

     make -j$(nproc)

is a better alternative?